src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -117,11 +112,27 @@ def _handle_result(cmd, result_poller, source_registry, source_image, registry):
                         registry = get_registry_from_name_or_login_server(cmd.cli_ctx, source_registry, source_registry)
 
                 if registry.login_server.lower() in source_image.lower():
-                    logger.warning("Import from source failed.\n\tsource image: '%s'", "{}/{}"
+                    logger.warning("Import from source failed.\n\tsource image: '%s'\n"
+                                   "Attention: When source registry is specified with --registry, "
+                                   "--source is considered to be a source image name. ", "{}/{}"
```

Comment:
Nice update.

Consider adding something like
```suggestion
                                   "--source is considered to be a source image name.  Do not prefix `--source` with the registry login server name.", "{}/{}"
```